custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -72,6 +72,27 @@ def single_value_conditions(cls) -> List["Conditions"]:
             cls.NOT_LESS_THAN,
         ]
 
+    @classmethod
```

Comment:
these added class methods aren't really useful in any way, they are _only_ for _a_ uniti test suite.  can we just make them constants within the unit tests, please.  having them here, will just confuse any future users. 

also, this isn't numeric conditions, it's `ONLY_FOR_NUMERIC`

Reply:
if they need to be imported, maybe make a constants file in the unit test directory?

Reply:
DONE